### PR TITLE
Simplify installation docs

### PR DESCRIPTION
**Pull request by pcoet**
This PR simplifies the installation instructions for the example chat app and also removes an install script that was installing Node/Python on the user's machine. (We should let users do that for themselves.) Almost all of the changes to the table are just reformatting for readability in markdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,247 +21,310 @@
...
+
+* A Python [Flask](https://flask.palletsprojects.com/en/3.0.x/quickstart/)
+  server, which demonstrates the
+  [Gemini API Python SDK](https://github.com/google-gemini/generative-ai-python)
+* A Node.js server, which demonstrates the
+  [Gemini API JavaScript SDK](https://github.com/google-gemini/generative-ai-js).
+* A Go server, which demonstrates the
+  [Gemini API Go SDK](https://github.com/google/generative-ai-go) 
 
 ## API documentation
```
Comment (markmcd) [inline README.md:52]: Just a thought - would this be better if they were moved after the setup steps? Seems like the value of the app is understanding & running, not necessarily providing a layer of indirection in front of the real API.

File: README.md [hunk 2]
```diff
@@ -24,247 +21,310 @@
...
+
 ```
 go version
 ```
-If Go is not installed, follow the instructions for your operating system from the [official Go installation guide](https://go.dev/doc/install).
+
+If Go is not installed, follow the instructions for your operating system from
+the [Go installation guide](https://go.dev/doc/install).
 
 ## Run the app
```
Comment (markmcd) [inline README.md:213]: The sub-headings below this one are hard to distinguish - I think that's adding to my confusion about which is required / optional.  e.g. when reading through Node, I tried to skim down to "Run the app" but there are three "run the app" sections the follow.

Could they go in separate setup files? Or maybe [collapsible sections](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)?

**Review by markmcd — COMMENTED** [review]
Unrelated to your change, but the top-line heading for this can't be "Gemini chat app" - it confuses the branding between the "Gemini chat app" that lives on gemini.google.com (which is already very confusing). The images at the top are misleading too - I don't think they would pass marketing approval.

And +1 to removing the language setup. If we want to provide that level of quick-setup we can provide a docker image or something instead.

File: README.md (hunk 2 above)
Comment (pcoet) [reply README.md:213]: I added a comment that hopefully clarifies things...

File: README.md [hunk 3]
```diff
@@ -24,247 +21,310 @@
...
+          <td>200</td>
+          <td>application/json</td>
+          <td>string</td>
+        </tr>
+      </table>
+    </td>
+  </tr>
 </table>
 
 ## Installation
```
[the remark this answers is not in the document]
Comment (pcoet) [reply README.md:52]: Ah, just saw this. I think with the updates to the README this should be more clear, but let me know if you still have concerns.

**Review by markmcd — APPROVED** [review]
Awesome thank you for the updates.

**Pull request merged**